### ISSUE TITLE
Facenet on Android

### ISSUE THREAD

**Issue by AdamMiltonBarker** [davidsandberg/facenet#818]
Hi guys does anyone know of any examples of using Tensorflow Facenet on Android? Have been working on TensorFlowInferenceInterface but have not had success yet.

**Comment by renxudeji** [comment]
It is difficult!

**Comment by AdamMiltonBarker** [comment]
Thanks I realize that :) Am making a tutorial for it if anyone has any pointers I literraly have found no helpful info :(

**Comment by speculaas** [comment]
Dear All, 
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android

I guess everybody probably know the above? (as a good starting point?)

Helplessly staring at all these android details?

Don't you think working in python-land is a blessing? Relatively.

Android? Aosp kingdom? Like trying to run in water? so much resistance?

BR,
Jimmy

**Comment by speculaas** [comment]
Dear All, 
Maybe this can help?
https://github.com/davidsandberg/facenet/issues/432
BR,
Jimmy

**Comment by AdamMiltonBarker** [comment]
That is similar to what I have been working on but not got there yet.

**Comment by arpitworld** [comment]
Hi - Was anyone able to achieve this yet?

**Comment by speculaas** [comment]
I think I maybe did achieve a non crashing apk
but the android logits are different from python logits
Still cannot find why
(I want logit instead of embedding so I did some modification)
BR,
Jimmy
